eoscostarica/lifebank
# fix(frontend): solve is define or is assigned but never use issues

Resolves #838 

### What does this PR do?
Solve is defined or is assigned but never use issues

### How should this be tested?
- Make run
- Check there is no console log errors with defined or assigned status
